app/controllers/gkr_keyword_comparison_controller.rb

```diff
@@ -0,0 +1,10 @@
+class GkrKeywordComparisonController < ApplicationController
+  def show
+  end
+
+  def create
+    concept_id = params[:concept_id]
+    threshold = params[:threshold]
+    collection_hash = Cmr.get_collection(concept_id, 'umm_json') # using this method bc get_concept() did not work for all concept ids
```

Comment:
Do you want to make collection_hash available to the view, if so you can put a @ in front of it.  Maybe also just rename it to @collection

Reply:
Then setup a page that renders the abstract and science keywords maybe.